api/provider/aws/load_balancer_create.go

```diff
@@ -219,13 +219,8 @@ func (l *LoadBalancerProvider) portsToListeners(ports []models.Port) ([]*elb.Lis
 		listener.SetLoadBalancerPort(port.HostPort)
 		listener.SetInstancePort(port.ContainerPort)
 
-		if port.CertificateName != "" {
-			certificateARN, err := l.lookupCertificateARN(port.CertificateName)
-			if err != nil {
-				return nil, err
-			}
-
-			listener.SetSSLCertificateId(certificateARN)
+		if certificate := port.CertificateARN; certificate != "" {
```

Comment:
we don't need this variable anymore, this can just be 
```go
if port.CertificateARN != "" {
    listener.SetSSLCertificateId(port.CertificateARN)
}
```